src/main/java/uk/gov/hmcts/reform/sscs/services/DataMigrationService.java

```diff
@@ -19,23 +19,21 @@ public class DataMigrationService {
     private final CcdCasesSender ccdCasesSender;
     private final IdamService idamService;
 
-    @Value("${features.data-migration.encoded-data-string}")
-    private String encodedDataString;
-
     public DataMigrationService(CcdCasesSender ccdCasesSender, IdamService idamService) {
         this.ccdCasesSender = ccdCasesSender;
         this.idamService = idamService;
     }
 
-    public void process(String languageColumn) throws IOException {
+    public void process(String migrationColumn, DataMigrationJob job, String encodedDataString) throws IOException {
         JSONArray data = new JSONArray(decompressAndB64Decode(encodedDataString));
         AtomicInteger unprocessed = new AtomicInteger(data.length());
         log.info("Number of cases to be migrated: ({})", unprocessed.get());
         data.iterator().forEachRemaining(row -> {
-            boolean success = ccdCasesSender.updateLanguage(
+            boolean success = ccdCasesSender.updateCaseMigration(
```

Comment:
would updateCase/updateCaseData be more accurate?

Reply:
Yeah, issue is there is already a method called updateCase in ccdCaseSender being used as part of the case loader service.